codelists/views/version_diff.py

```diff
@@ -12,17 +11,34 @@
 
 @load_version
 def version_diff(request, clv, other_tag_or_hash):
-    q = Q(tag=other_tag_or_hash)
+    other_clv = None
+
+    # 1. If other_tag_or_hash is a hash, try to find the CodelistVersion by ID
     try:
         id = unhash(other_tag_or_hash, "CodelistVersion")
+        qs = CodelistVersion.objects.filter(id=id)
```

Comment:
micronit: I'm guessing `qs` == "QuerySet", I'd be inclined to go for a more meaningful variable name